Transcribe89

gregmacfarlane opened the pull request:
This is the pull request for the Chapter 8 and 9 work

barnesmaxmax committed:
Add Chapter 6 Text

Added text for Chapter 6.

Commit:
Add eq 6.5 -6.10 and unlisted eq in between

gregmacfarlane committed:
Add chapter 6 setup chunk

gregmacfarlane committed:
Model segmentation example

loganbennett93 committed:
Begin Tables 6-9 and 6-10

I have the tables set up the way I want for the most part, but the values are a little off from what appears in the text. I also attempted to create in-text references, though I haven't figured it out completely yet.

camlunt committed:
code block for table 6-12

camlunt committed:
Functions for Value of Time (VOT)

camlunt committed:
First draft of Model 16W

camlunt committed:
Create new columns Travel Time

Includes TT for motorized modes only, TT for non-motorized modes only, and OVTT for motorized modes only.

camlunt committed:
Update on Model 16W

Adjusted the travel time variables

camlunt committed:
Update of Model 16W (still not quite right though)

camlunt committed:
No need for new columns

camlunt committed:
Model 17W and Code block for table 6-13

camlunt committed:
Code block for Table 6-14

camlunt committed:
Code block for Table 6-15

Commit:
add basic info for table 6.1, 6.3, and 6.8

barnesmaxmax committed:
Add in text math and base of table 6-5

barnesmaxmax committed:
Equation changes for table 6-5

loganbennett93 committed:
Add Table 6-11 and update Table 6-9

I've incorporated the pertinent needed for models 11w, 13w, 14w, 15w, and 16w, but even though they show up in the tables as in the text, the values are still slightly off from those in the text.

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

gregmacfarlane committed:
Just read in the data, don't need to remake it.

gregmacfarlane committed:
Commit modified worktrips file

gregmacfarlane committed:
Already read this file in above

Commit:
adjust tables

Commit:
Merge branch 'transcribe6' of https://github.com/byu-transpolab/modechoice into transcribe6

Commit:
update tables

Commit:
tables update

loganbennett93 committed:
Update Models 11w, 13w, 14w, 15w, 16w

These models appear in Table 6-9 and 6-11. I mutated the nm_ovtt column to divide by the distance. The values are closer to what appear in the text, but still slightly off.

Commit:
fix table 6-8

Commit:
Merge branch 'transcribe6' of https://github.com/byu-transpolab/modechoice into transcribe6

camlunt committed:
Update on VOT functions

loganbennett93 committed:
Update Model 11-w

We will need to change the constrained values in other models to equal the values given in the text

camlunt committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

loganbennett93 committed:
Minor changes to models and remove constraints

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

Commit:
adjust table 6.1, 6.3, and 6.8. w/out setting shared ride 2 = 3+

bwang0601 committed:
merge

SamKLau committed:
Equations 6.1 to 6.3

SamKLau committed:
Equation committing

Merge branch 'transcribe6' of github.com:byu-transpolab/modechoice into transcribe6

# Conflicts:
#	06-specification.Rmd

SamKLau committed:
chapter 6 footnotes

SamKLau committed:
equations 6.1-6.4

SamKLau committed:
start of table 6-6 and 6-7

loganbennett93 committed:
Update Table 6-10

This table now gives the correct values of VoT with coefficients drawn straight from the models. However, the model coefficients are still a little off

camlunt committed:
Table 6-12

The first column is calculated from model coefficients. The second column uses hard-coded numbers. I can't figure out where the 0.47, 0.77, and 0.62 come from.

camlunt committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

loganbennett93 committed:
Begin Table 6-4

barnesmaxmax committed:
Fix Table 6-5

Table 6-5 has correct values (or as correct as possible).

barnesmaxmax committed:
Change duplicated chunk name

camlunt committed:
Commented out later version of VOT equations

camlunt committed:
Model 17W

camlunt committed:
Segmentation by auto ownership

camlunt committed:
Segmentation by gender

Using femdum doesn't result in the correct parameters, but it's the best option I've come up with so far.

barnesmaxmax committed:
Update Table 5-8 Code

camlunt committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

barnesmaxmax committed:
Merge branch 'transcribe5' into transcribe6

barnesmaxmax committed:
Add coefficient name fixes in table 6-1

loganbennett93 committed:
Auto stash before merge of "transcribe6" and "origin/transcribe6"

loganbennett93 committed:
Add Table 6-4

camlunt committed:
Added constraints and filters to Model 17W and the segmented models

camlunt committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

camlunt committed:
Fixed stashing issue for Table 6-4

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

loganbennett93 committed:
Fix Table 6-4

SamKLau committed:
Table 6-4

loganbennett93 committed:
Add in-text chapter/section references

I updated only those headings that were called out in Chapter 6, with a simple notation of chapters and sections followed by their respective numbers

barnesmaxmax committed:
Start Table 6-2

barnesmaxmax committed:
Add partial table 6-2

gregmacfarlane committed:
Merge branch 'transcribe5' into transcribe6

camlunt committed:
Table 6-12

gregmacfarlane committed:
Provide basic estimation of nested logit models

gregmacfarlane committed:
Build doc

gregmacfarlane committed:
Build table for model 7 values of time

loganbennett93 committed:
Add Chapter 8 text

loganbennett93 committed:
Add Chapter 9 text

gregmacfarlane committed:
Edit income interaction table

barnesmaxmax commented:
Max:
Chapter 8:
- [ ] Equations 8.1-8.8
- [ ] Equations 8.18-8.21
Chapter 9:
- [ ] Figure 9.1
- [ ] Table 9.1
- [ ] Table 9.6

Camille:
Chapter 8:
- [ ] Equations 8.9-8.12
- [ ] Table 8.2
Chapter 9:
- [ ] Figure 9.4
- [ ] Table 9.4
- [ ] Figure 9.6
- [ ] Table 9.7

Sami:
Chapter 8:
- [ ] Table 8.1
- [ ] Equations 8.13-8.17
Chapter 9:
- [ ] Figure 9.3
- [ ] Table 9.3
- [ ] Table 9.5 (models already made in Table 9.3)
- [ ] Table 9.9

Bruce:
Chapter 8:
- [ ] Figures 8.1-8.4
- [ ] Equations 8.31-8.33
Chapter 9:
- [ ] Figure 9.2
- [ ] Table 9.2
- [ ] Figure 9.5
- [ ] Table 9.8

Logan:
Chapter 8:
- [ ] In-text references
- [ ] In-text greek notation
- [ ] Table 8.3
- [ ] Equations 8.22-8.30
Chapter 9:
- [ ] In-text references
- [ ] In-text greek notation
- [ ] Help with other tasks as needed

barnesmaxmax committed:
Add changes to table 6.2 and figure 6.1

Add changes in an attempt to get these items to show correct information.

barnesmaxmax committed:
Add equation 8.1

barnesmaxmax committed:
Add changes to table 6-2 and table 6-1

SamKLau committed:
merged commits with previous branch

Merge branch 'transcribe6' of github.com:byu-transpolab/modechoice into transcribe6

# Conflicts:
#	06-specification.Rmd

SamKLau committed:
merge from previous requests

SamKLau committed:
updated equations

SamKLau committed:
adjustments to equations 6.4 and 6.5

SamKLau committed:
Tables 6.6 and Tables 6.7

Commit:
add figures 8.1-8.4

SamKLau committed:
updates to equations 6.5 and 6.6, tables 6.6 and 6.7

Commit:
add equations 8.31-8.33

gregmacfarlane committed:
Start estimation of Table 9.1

gregmacfarlane committed:
Remove old nesting examples

gregmacfarlane committed:
Add likelihood ratio test for table 9.1

gregmacfarlane committed:
Start parallel nests chunk

gregmacfarlane committed:
Add parallel nests models

SamKLau committed:
insert r code for table 8.1

SamKLau committed:
Updated equations 8.13 to 8.17

Commit:
model 24w

bwang0601 committed:
merge

Commit:
add figure 9.2 and 9.5

Commit:
add table 9.8

Commit:
add tabel9.8

camlunt committed:
Format changes to Table 6-12

camlunt committed:
Fixed Table 6-13

camlunt committed:
Added caption to Table 6-14

camlunt committed:
Fixed formatting in bullet points

camlunt committed:
Added caption for Table 6-15

gregmacfarlane committed:
Clean up references in first half of chapter 6

gregmacfarlane committed:
Delete unnecessary intermediate build files

loganbennett93 committed:
Add in-text references and Greek notation to Chapters 8 and 9

I updated sections references in some other chapters to be able to include them here

gregmacfarlane committed:
Clean up multiline equations in chapter 6

loganbennett93 committed:
Add equations 8.22-8.30

camlunt committed:
Equations 8.9-8.12

camlunt committed:
Table 8-2 - most of it

Most of what's needed is here. Still need to fix the row headers (merge calls) and add table caption.

camlunt committed:
Merge remote-tracking branch 'origin/transcribe89' into transcribe89

camlunt committed:
Add equations 8.11 and 8.12 back in

They got removed in the resolution of some merge conflicts I performed.

camlunt committed:
Added four figures to the img folder

camlunt committed:
Changed file structure to figures

gregmacfarlane committed:
Add example of using the apollo library for mode choice

gregmacfarlane committed:
More edits to the apollo demos

loganbennett93 committed:
Add Table 8-3 and in-text reference

barnesmaxmax committed:
Add equations 8.2-8.8 and 8.18-8.21

camlunt committed:
Added figures and fixed some bugs

camlunt committed:
Saved figures to the img folder

barnesmaxmax committed:
Add image 9-1

barnesmaxmax committed:
Add table 9.6

gregmacfarlane committed:
Merge remote-tracking branch 'origin/transcribe89' into transcribe89

barnesmaxmax committed:
Add Shop/Other trips

Adding new data file for shop/other trips and saved .rds file.

barnesmaxmax committed:
Add changes to models and input data

Table 9-6 works, though it isn't complete nor accurate. I'm not sure what I can do to fix it.

gregmacfarlane committed:
Merge remote-tracking branch 'origin/transcribe89' into transcribe89

SamKLau committed:
equation chnages

SamKLau committed:
merging old branch with new

Merge branch 'transcribe89' of github.com:byu-transpolab/modechoice into transcribe89

# Conflicts:
#	08-nestedlogit.Rmd

gregmacfarlane committed:
Fix cacheing and other page options

gregmacfarlane committed:
Fix some equation labels

gregmacfarlane committed:
Fix ben akiva reference

gregmacfarlane committed:
Clean up code in vot ratio figure

gregmacfarlane committed:
Build site

gregmacfarlane committed:
Merge branch 'transcribe6' into transcribe89

gregmacfarlane committed:
Add nesting images

gregmacfarlane committed:
Revise and make shopother data

gregmacfarlane committed:
Make sure that images are there

gregmacfarlane committed:
Don't need libs in top folder

gregmacfarlane committed:
Minor changes for build

gregmacfarlane committed:
Build
